client/signup/steps/business-needs/index.jsx

```diff
@@ -0,0 +1,190 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import { isUserLoggedIn } from 'state/current-user/selectors';
+
+//Form components
+import Card from 'components/card';
+import Button from 'components/button';
+import FormInputCheckbox from 'components/forms/form-checkbox';
+import FormLabel from 'components/forms/form-label';
+import FormFieldset from 'components/forms/form-fieldset';
+
+class BusinessNeeds extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			businessNeeds: get( this.props, 'signupProgress[2].providedDependencies.businessNeeds', '' ),
+		};
+	}
+
+	checkBoxHandleChange = event => {
+		const fieldValue = this.state.businessNeeds;
+		const valuesArray = fieldValue ? fieldValue.split( ',' ) : [];
+
+		if ( valuesArray.indexOf( event.target.value ) === -1 ) {
+			valuesArray.push( event.target.value );
+		} else {
+			valuesArray.splice( valuesArray.indexOf( event.target.value ), 1 );
+		}
+
+		this.setState( { businessNeeds: valuesArray.join( ',' ) } );
+	};
+
+	isCheckBoxChecked( goal ) {
+		//const fieldValue = this.state.businessNeeds;
+		const initialVal = this.state.businessNeeds.split( ',' );
+		if ( initialVal.indexOf( goal ) !== -1 ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	handleSubmit = event => {
+		event.preventDefault();
+		const { goToNextStep, stepName, flowName, translate } = this.props;
+
+		//Create site
+		SignupActions.submitSignupStep(
+			{
+				processingMessage: translate( 'Collecting your information' ),
+				stepName: stepName,
+			},
+			[],
+			{
+				businessNeeds: this.state.businessNeeds,
+			}
+		);
+
+		goToNextStep( flowName );
+	};
+
+	renderNeedsCheckboxes() {
+		const { translate } = this.props;
+		// Note that the key attributes will be used in the name of a tracks event attribute so can not
+		// contain whitespace.
+		const options = {
+			contact: {
+				key: 'contact',
+				formLabel: translate( 'Allow people to contact me' ),
+			},
+			payments: {
+				key: 'payments',
+				formLabel: translate( 'Collect payments' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Collect Payments and Donations' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).